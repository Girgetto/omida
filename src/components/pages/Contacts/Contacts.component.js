import React from 'react'
import { useTranslation } from 'react-i18next'
import { SEO } from '../../layout'
import contact from '../../../assets/img/contact.jpg'
import styles from '../Home/components/style.module.scss'

const Contacts = ({ className }) => {
  const [t] = useTranslation()
  return (
    <div className={className}>
      <SEO title="Omida - Real Estate Consulting - Italy |Â Contacts" />
      <div>
        <p>{t('contact.ContactUs', 'Contacts')}</p>
        <div>
          <a href="mailto:omida@omida-realestate.com">
            <b>Email:</b> omida@omida-realestate.com
          </a>
          <div className={styles.dog}>
            <img className={styles.image} src={contact} alt="contact" />
          </div>
        </div>
      </div>
    </div>
  )
}

export default Contacts
