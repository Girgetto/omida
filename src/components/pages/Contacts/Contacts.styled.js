import styled from 'styled-components'

import Contacts from './Contacts.component'

export default styled(Contacts).attrs()`
    width: 80%;
    margin: 0 auto;
    display: flex;
    justify-content: center;
    align-items: center;
    min-height: 80vh;

    a {
      text-decoration: none;
      color: #fff;
    }
  }
`
