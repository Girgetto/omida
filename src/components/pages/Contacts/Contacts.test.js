import React from 'react';
import { render } from '@testing-library/react';
import { ParallaxProvider } from 'react-scroll-parallax'

import Contacts from './Contacts.component';

const props = {
  className: 'className',
};

describe('<Contacts />', () => {
  const { container } = render(
    <ParallaxProvider>
      <Contacts {...props} />,
    </ParallaxProvider>
  );

  it('match snapshot', () => {
    expect(container).toMatchSnapshot();
  });
});
