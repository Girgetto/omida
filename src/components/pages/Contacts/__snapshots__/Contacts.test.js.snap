// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contacts /> match snapshot 1`] = `
<div>
  <div
    class="className"
  >
    <div>
      <p>
        contact.ContactUs
      </p>
      <div>
        <a
          href="mailto:omida@omida-realestate.com"
        >
          <b>
            Email:
          </b>
           omida@omida-realestate.com
        </a>
        <div
          class="dog"
        >
          <img
            alt="contact"
            class="image"
            src="contact.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  ,
</div>
`;
